# exceptions
err.system.error=An internal error has occurred. Reference id: ''{0}''